backend/curation/api/curation-api.yml

```diff
@@ -511,34 +538,141 @@ paths:
         "401":
           $ref: "#/components/responses/401"
 
-  /v1/collections/{collection_id}/datasets/{dataset_id}/assets:
+#  /v1/collections/{collection_id}/datasets/{dataset_id}/assets:
+#    get:
+#      summary: Retrieve links for downloading a Dataset
+#      tags:
+#        - collection
+#      description: |
+#        Generate Presigned S3 URLs to download the files associated with the `dataset_id` in the Collection specified
+#        by the `collection_id`. Pre-signed S3 URLs have a life span of up to 1 hour. You should generate a pre-signed
+#        URL immediately before each download to ensure it is current.
+#      operationId: backend.curation.api.v1.curation.collections.collection_id.datasets.dataset_id.assets.get
+#      parameters:
+#        - $ref: "#/components/parameters/path_collection_id"
+#        - $ref: "#/components/parameters/path_dataset_id"
+#      responses:
+#        "200":
+#          description: OK
+#          content:
+#            application/json:
+#              schema:
+#                $ref: "#/components/schemas/asset_response"
+#        "202":
+#          description: Unable to retrieve one or more asset response fields.
+#          content:
+#            application/json:
+#              schema:
+#                $ref: "#/components/schemas/asset_response"
+#        "400":
+#          $ref: "#/components/responses/400"
+#        "404":
+#          $ref: "#/components/responses/404"
+
+  /v1/datasets:
     get:
-      summary: Retrieve links for downloading a Dataset
+      summary: List Cell Census previews of all Datasets
```

Comment:
"Cell Census" must not appear in the swagger. 

Cell Census is only one example of a bulk downloader. The solution needs to be general purpose.